addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
chiming in here. I haven't seen scroll be synchronous, just pulled this from putting a breakpoint [here](https://github.com/alphasights/ember-scrollable/compare/master...alexander-alvarez:testing-wait#diff-bcc853db8ecac305500add8bb7a0f313R164) and running tests in Chrome
![image](https://cloud.githubusercontent.com/assets/11779993/25351761/ae2fc378-28f7-11e7-9468-bd0bd2690bf6.png)